Code/src/main/java/nl/utwente/group10/ui/components/blocks/FunctionBlock.java

```diff
@@ -327,4 +352,20 @@ public final void error() {
     public OutputAnchor getOutputAnchor() {
         return output;
     }
+
+    @Override
+    public void changed(ObservableValue<? extends Number> arg0, Number arg1, Number arg2) {
+        if (arg0.equals(bowtieIndex)) {
+            invalidateBowtieIndex();
+        }
+    }
+
+    private void invalidateBowtieIndex() {
+        for (InputAnchor input : getAllInputs()) {
+            input.setVisible(getInputIndex(input) < getBowtieIndex());
+            if(input.isConnected()){
```

Comment:
Hmm. Ik bedoelde specifiek de alle-inputs-loskoppelen behaviour. Dit is wat ik krijg als ik de bowtie index aanpas:

![2015-05-08-214515_318x329_scrot](https://cloud.githubusercontent.com/assets/126832/7544217/746f7176-f5cb-11e4-8dce-6cf77983e801.png)